### PR TITLE
Fixed praetor startime on leaders and followers.

### PR DESCRIPTION
**Pull request by jeremykross**
@nicktate @normanjoyner

### CODE REVIEW

**Comment by nicktate** [comment]
we shouldn't even return this `praetor` info. if the frontend has any issues we will add logic to handle it there.

**Comment by nicktate** [comment]
just rip out `praetor.start_time`

File: lib/kubernetes-api.js [hunk 1]
```diff
@@ -588,6 +590,16 @@ class KubernetesApi extends ApiImplementation {
                         return cb(null, util.safeParse(body));
                     });
             },
+            services: (cb) => {
+                return request(`${this.k8sApiUrl}/api/v1/serivces`,
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: `services`

**Comment by jeremykross** [comment]
@nicktate  or @normanjoyner - do we store the start time elsewhere on the node?

**Comment by jeremykross** [comment]
updated, matched the follower, placing it on the host root.

**Pull request merged**